docs/resources/xopp-to-pdf

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+SCRIPT_TITLE=xopp-to-pdf
```

Comment:
I would modify the title to be somewhat nicer to read.
```suggestion
SCRIPT_TITLE=Convert *.xopp to *.pdf
```

Reply:
Good idea, I'll add it with single quotes.